demo/src/Header.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { Box, Link, Typography } from "@mui/material";
+import image from "../../assets/logo_square.png";
+
+export default function Header() {
+    return (
+        <Box
+            sx={{
+                display: "flex",
+                justifyContent: "space-between",
+                alignItems: "center",
+                mb: "1rem",
+            }}
+        >
+            <Box>
+                <Typography
```

Comment:
It's good to use the `Typography` element. The variants, however, should follow HTML semantics. So the title should be `h1`, and the text underneath should be `subtitle1`.

Reply:
We noticed that `<Typography variant="h1">` and `<h1>` result in very different font sizes. Which would you like us to use (screenshots provided below)? Furthermore, if we are to use `<h1>` tags, what subtitle tag would you like us to use? For reference, Jerry used `<p>` tags in `index.html`.

We are also happy to move everything back to `index.html` and keep it the way it previously was.

Note: The extra margin bottom for the `<h1>` component is added by `picocss`.

<details>
<summary>Using Typography component</summary>

<img width="959" alt="Screenshot 2024-08-14 231416" src="https://github.com/user-attachments/assets/07dc35d5-a53f-4731-96bb-9103cf3e1cad">

</details>

<details>
<summary>Using h1 component</summary>

<img width="951" alt="Screenshot 2024-08-14 231637" src="https://github.com/user-attachments/assets/2e456a38-fc99-4f45-8533-7adcd0b4373f">

</details>